frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/report/simulated-region-overview-behavior-report.component.ts

```diff
@@ -0,0 +1,135 @@
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type {
+    RecurringEventActivityState,
+    ReportableInformation,
+    ReportBehaviorState,
+} from 'digital-fuesim-manv-shared';
+import { reportableInformations, UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import { Subject, combineLatest, map } from 'rxjs';
+import { ExerciseService } from 'src/app/core/exercise.service';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    createSelectActivityStates,
+    createSelectBehaviorState,
+} from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-simulated-region-overview-behavior-report',
+    templateUrl: './simulated-region-overview-behavior-report.component.html',
+    styleUrls: ['./simulated-region-overview-behavior-report.component.scss'],
+})
+export class SimulatedRegionOverviewBehaviorReportComponent
+    implements OnInit, OnDestroy
+{
+    @Input() simulatedRegionId!: UUID;
+    @Input() reportBehaviorId!: UUID;
+
+    private readonly destroy$ = new Subject<void>();
+
+    reportBehaviorState$!: Observable<ReportBehaviorState>;
+
+    activities$!: Observable<{
+        [key in ReportableInformation]?: RecurringEventActivityState;
+    }>;
+
+    reportableInformations = reportableInformations;
+    reportableInformationTranslationMap: {
+        [key in ReportableInformation]: string;
+    } = {
+        patientCount: 'Anzahl an Patienten',
+        vehicleCount: 'Anzahl an Fahrzeugen',
+        personnelCount: 'Anzahl an Rettungskräften',
+        materialCount: 'Anzahl an Material',
+        treatmentStatus: 'Behandlungsstatus',
+    };
+
+    createReportCollapsed = true;
+    selectedInformation: ReportableInformation | 'noSelect' = 'noSelect';
+
+    constructor(
+        private readonly exerciseService: ExerciseService,
+        private readonly store: Store<AppState>
+    ) {}
+
+    ngOnInit(): void {
+        this.reportBehaviorState$ = this.store.select(
+            createSelectBehaviorState(
+                this.simulatedRegionId,
+                this.reportBehaviorId
+            )
+        );
+
+        const activities$ = this.store.select(
```

Comment:
Consider renaming this variable or `this.activities$` to avoid confusion between them.